scipy/stats/tests/test_continuous_basic.py

```diff
@@ -196,7 +198,12 @@ def test_cont_basic(distname, arg, sn, n_fit_samples):
 
     check_named_args(distfn, x, arg, locscale_defaults, meths)
     check_random_state_property(distfn, arg)
-    check_pickling(distfn, arg)
+
+    if distfn in ['rel_breitwigner'] and _IS_32BIT:
```

Comment:
It turns out this is using the wrong variable.  `distfn` is the object; `distname` should be used here.

Reply:
Can you make a PR?

Reply:
Yup, I'm just about to click the button on it...

Reply:
https://github.com/scipy/scipy/pull/18493